### PR TITLE
Sprint 1 Pull Request Fixed

### PR DESCRIPTION
**Pull request by vumd-UC**
Original pull request with some code review comments is [here](https://github.com/moburke/WhatShouldIRead/pull/1)
Original pull request got preemptively closed, so to allow for everyone to view what was changed and to allow for a full code review, this pull request was created.

### CODE REVIEW

**Comment by vumd-UC** [comment]
Original pull request with some code review comments is here  [https://github.com/moburke/WhatShouldIRead/pull/1]
Original pull request got preemptively closed, so to allow for everyone to view what was changed and to allow for a full code review, this pull request was created.

File: app/src/main/java/com/example/whatshouldiread/customclasses/Book.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
+package com.example.whatshouldiread.customclasses;
+
+/*
+ * Class that stores information for a particular book. Allows for links to amazon for the particular
+ * book be created through a function, rather than hardcoding.
+ */
+
```
Comment (vumd-UC) [inline app/src/main/java/com/example/whatshouldiread/customclasses/Book.java:7]: @petromw Updated the Javadocs for the files based on the comments from the original pull request. Please resolve this comment when you're okay with these changes.

File: app/src/main/java/com/example/whatshouldiread/customclasses/Question.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package com.example.whatshouldiread.customclasses;
+
+/*
+ * Class to store a question in text form as well as what position the question will be. Allows for
+ * multiple questions for the same position without changing the end result of the books.
+ */
+
```
Comment (vumd-UC) [inline app/src/main/java/com/example/whatshouldiread/customclasses/Question.java:7]: @petromw Updated the Javadocs for the files based on the comments from the original pull request. Please resolve this comment when you're okay with these changes.

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <AndroidXmlCodeStyleSettings>
+      <option name="ARRANGEMENT_SETTINGS_MIGRATED_TO_191" value="true" />
+    </AndroidXmlCodeStyleSettings>
```
Comment (vumd-UC) [inline .idea/codeStyles/Project.xml]: test

**Review by moburke — APPROVED** [review]
Looks like we're good to go

**Review by petromw — APPROVED** [review]
Looks good to me

File: app/src/main/java/com/example/whatshouldiread/ui/home/HomeFragment.kt [hunk 4]
```diff
@@ -22,10 +22,7 @@ class HomeFragment : Fragment() {
         homeViewModel =
             ViewModelProviders.of(this).get(HomeViewModel::class.java)
         val root = inflater.inflate(R.layout.fragment_home, container, false)
-        val textView: TextView = root.findViewById(R.id.text_home)
-        homeViewModel.text.observe(this, Observer {
-            textView.text = it
-        })
+        homeViewModel.text.observe(viewLifecycleOwner, Observer {})
```
Comment (discospiff) [inline app/src/main/java/com/example/whatshouldiread/ui/home/HomeFragment.kt:25]: It looks like this change removed some functionality.

**Pull request merged**